src/elements/hx-search-control/index.spec.js

```diff
@@ -0,0 +1,98 @@
+import { fixture, expect, oneEvent } from '@open-wc/testing';
+
+/**
+ * <hx-search-control> component tests
+ *
+ * @type HXSearchControlElement
+ *
+ */
+describe('<hx-search-control> component tests', () => {
```

Comment:
Please add a test for the `controlElement` method:

```js
    get controlElement () {
        return this.querySelector('input[type="search"]');
    }
```

As we discussed today, you can get the control element value using:

```js
const ctrl = fragment.controlElement;
```

Reply:
Changes have been done, please review.